hedera-node/src/main/java/com/hedera/services/store/models/TokenRelationship.java

```diff
@@ -59,6 +60,11 @@ public TokenRelationship(Token token, Account account) {
 		this.account = account;
 	}
 
+	public void revokeKyc() {
+		validateTrue(token.hasKycKey(), TOKEN_HAS_NO_KYC_KEY);
+		kycGranted = false;
+	}
+
```

Comment:
Can we think of a more clear design of the feature? 
I'm thinking of the following:
We can use 1 function in both transition logics - `changeKyc` or `updateKyc` which takes an input arg of `boolean` type.